src/makeCalculator.js

```diff
@@ -4,7 +4,44 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+    add: function (value) {
```

Comment:
```suggestion
    add(value) {
```
we can simplify that

Reply:
Thanks! now that's how I'll write ;)